rtl/ecltpl/thortpl.cpp

```diff
@@ -16,10 +16,10 @@
 ############################################################################## */
 
 $?$/* Template for generating thor/hthor/roxie output */
-#if defined(__clang__) || (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 2))
-#pragma GCC diagnostic ignored "-Wall"
-#pragma GCC diagnostic ignored "-Wextra"
-#endif
+//#if defined(__clang__) || (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 2))
```

Comment:
Is this commented out deliberately?

Reply:
Not really. Was just trying to work out whether generated code was giving any warnings